ivy/functional/ivy/experimental/creation.py

```diff
@@ -33,9 +33,8 @@ def vorbis_window(
     dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None,
     out: Optional[ivy.Array] = None,
 ) -> ivy.Array:
-    """
-    Return an array that contains a vorbis power complementary window of size
-    window_length.
+    """Return an array that contains a vorbis power complementary window of
+    size window_length.
```

Comment:
Make sure you are not modifying any existing code in the file. It looks like you've reformatted a lot of the docstrigs, could you revert that change please?